gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemConfiguration.java

```diff
@@ -95,6 +96,17 @@
   public static final HadoopConfigurationProperty<String> DELEGATION_TOKEN_BINDING_CLASS =
       new HadoopConfigurationProperty<>("fs.gs.delegation.token.binding");
 
+  /**
+   * Configuration key for GCS delegation token instantiation strategy. SHARED means delegation
+   * tokens are shared across multiple instanced of Cloud storage file services . Default value:
+   * INSTANCE_PER_SERVICE
+   */
+  public static final HadoopConfigurationProperty<GCSTokenInstantiationStrategy>
+      GCS_TOKEN_INSTANTION_STRATEGY =
```

Comment:
misspelled "INSTANTIATION".

Also, I suggest we still put "delegation" in the name. so:
"DELEGATION_TOKEN_INSTANTIATION_STRATEGY"